portfolio/src/main/webapp/index.html

```diff
@@ -57,14 +56,18 @@ <h3 id="header"><i>List of projects</i> &#128195;:</h3>
       <hr>
       <h3 id="header"><i>Contact me</i> &#128233;:</h3>
       <form action="/form-handler" method="POST">
-        <p>Enter some text here:</p>
-        <textarea name="text-input">hello world</textarea>
+        <p>Enter your email address:</p>
+        <textarea name="text-input">example@gmail.com</textarea>
         <br/>
         <input type="submit" />
       </form>
       <hr>
-      <h3 id="header"><i>Saying hello</i> &#128233;:</h3>
-      <p>Click <a href="/hello">here</a> to get a <strong>greeting</strong> from me! </p>
+      <h3 id="header"><i>Greetings</i> &#128075;:</h3>
+      <p>Click this button for a <strong>special</strong> message. </p>
+      <button onclick="displayGreeting();">
```

Comment:
Super nit: this doesn't need a semicolon after. Usually semicolons are used if you want to run two functions here.